### ISSUE TITLE
[Feature Request] Autoplay next file in folder.

### ISSUE THREAD

**Issue by derei** [abdallahmehiz/mpvKt#108]
I know there is a [Playlist feature request](https://github.com/abdallahmehiz/mpvKt/issues/8)  already in discussion, and this request would not only complement it, but also I'm guessing it would be much easier to implement: 

If the folder contains multiple videos (let's say all episodes of a season), it would be nice to have an option to automatically load next file.

**Comment by Sharad104** [comment]
yeah mpv-android does this like this ->
in file picker , we hold the folder which contains all the episode of a tv series and it automatically creates a playlist for us. this way not every folder becomes a playlist and feature is also neatly implemented .

**Comment by derei** [comment]
@Sharad104 when i attempted to olay the videos from a folder, it stopped after the first one. It didn't automatically go to next one.

**Comment by Sharad104** [comment]
> @Sharad104 when i attempted to olay the videos from a folder, it stopped after the first one. It didn't automatically go to next one.

I checked `mpv-android` and its autoloading the next episode after episode ends ( when opened folder via **holding** (not clicking) it via file picker )

I'm sure `mpv-kt` dev will definitely implement this feature better than `mpv-android` guys.

**Comment by Pentaphon** [comment]
> If the folder contains multiple videos (let's say all episodes of a season), it would be nice to have an option to automatically load next file.

I agree. I hate having to make playlists when I'm making folders from a video anyway.
